src/ngraph/codegen/compiler.cpp

```diff
@@ -227,8 +254,13 @@ void codegen::CompilerCore::initialize()
     auto LO = m_compiler->getInvocation().getLangOpts();
     LO->CPlusPlus = 1;
     LO->CPlusPlus11 = 1;
+#ifndef _WIN32
     // Strange but need to manually disable c++14
```

Comment:
I wonder if this is still true? Can we just enable c++14 for everything? Testing this would be difficult because CI does not currently test codegen

Reply:
@rkimballn1 I believe any changing for not Windows must be applied as a part of another commit and PR, in the current code the commit 'Strange but need to manually disable c++14' is already present. I think the reason could be the sized delete operator introduced in c++14, I had a lot of problems with its symbol on Windows (I have no idea why, but the symbol is not available for JITed code by default).